src/utils/azure.ts

```diff
@@ -12,7 +12,7 @@ import { createStorageClient } from './azureClients';
 import { nonNullProp, nonNullValue } from './nonNull';
 
 function parseResourceId(id: string): RegExpMatchArray {
-    const matches: RegExpMatchArray | null = id.match(/\/subscriptions\/(.*)\/resourceGroups\/(.*)\/providers\/(.*)\/(.*)/);
+    const matches: RegExpMatchArray | null = id.match(/\/subscriptions\/(.*)\/resourceGroups\/(.*)\/providers\/(.*)\/(.*)/) || id.match(/\/subscriptions\/(.*)\/resourcegroups\/(.*)\/providers\/(.*)\/(.*)/);
```

Comment:
Is casing the only difference in your regex?  If so, just add a "/i" tag to your regex to make it case-insensitive.

Reply:
ah good idea.

Reply:
I was not able to get the /i tag working. but, handles it in a different way

Reply:
Sorry just to be sure.  Did you try this? "/\/subscriptions\/(.*)\/resourceGroups\/(.*)\/providers\/(.*)\/(.*)/i"